### DISCUSSION TITLE
mirai-cpp项目编译后可执行文件显示网络错误(Executable file shows network error after mirai-cpp project is compiled )

### DISCUSSION THREAD

**Discussion by CodeMzt** [mamoe/mirai#1025] [Q&A]
刚接触mirai，打算用c++开发，选了mirai-cpp这个SDK
按照使用文档操作一番后，可以编译，并且生成了可执行文件，但可执行文件打开，输出网络错误（如图）
![image](https://user-images.githubusercontent.com/62409578/108046617-c7ca1800-707f-11eb-9b05-9c3ff52d192a.png)
导致插件无法运行，求助！
-----------------------------
I just got in touch with mirai and plan to develop in c++, so I chose mirai-cpp SDK
After operating according to the documentation, you can compile and generate an executable file, but the executable file is opened, and a network error is output (as shown in the figure)
![image](https://user-images.githubusercontent.com/62409578/108046617-c7ca1800-707f-11eb-9b05-9c3ff52d192a.png)
The plug-in cannot be run, please help!

**Comment by Karlatemp** [comment]
You should report to `mirai-cpp` not `mirai-core`

**Reply by Karlatemp** [reply]
社区SDK 问题与 mirai-core 本体没有关系

**Comment by cyanray** [comment]
你要先启动 mirai-console ([MCL](https://github.com/iTXTech/mirai-console-loader)), 并且配置好 [mirai-api-http](https://github.com/project-mirai/mirai-api-http)，才能使用基于 mirai-api-http 的插件。
